<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDemo21</class>
 <widget class="QDialog" name="dlgDemo21">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listar Dptos, Provs y Dist en QComboBox</string>
  </property>
  <widget class="QPushButton" name="btnVerUbigeo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver Ubigeo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboDepartamento">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cboProvincia">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cboDistrito">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblDepartamento">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona un Departamento</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProvincia">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona una Provincia</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDistrito">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona un Distrito</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
